app/views/references/_reference.html.erb

```diff
@@ -1,7 +1,29 @@
-<article class="p-3 mb-3 bg-light">
-  <h3 class="h4"><%= link_to reference.short_ref, reference %></h3>
-  <p class="mb-0 d-flex align-items-center">
-    <%= xr_icon Reference, {}, { class: "me-1", style: "height: 1.5rem;" } %>
-    <span><%= Reference.label.humanize %></span>
-  </p>
+<article class="card">
+
+  <div class="card-header">
+    <%= xr_icon Reference, {}, { class: "img-fluid", style: "width: auto; height: 1.1rem;" } %>
+    <%= Reference.label.humanize %>
+  </div>
+
+  <div class="card-body">
+    <h1 class="card-title h4">
```

Comment:
That might be a bit bold at times. Maybe more subtle/smaller?

Reply:
Yeah, I'd forgotten that in production `short_ref` is often not very short... how about as a quick fix we chop it off at 25 characters or so, pending adding `LONG_SHORT_REF` as an issue and cleaning the data (https://github.com/xronos-ch/xronos.rails/issues/260)?